crates/ruff/src/rules/flake8_pyi/rules/assignment_default_in_stub.rs

```diff
@@ -0,0 +1,17 @@
+use ruff_diagnostics::Violation;
+use ruff_macros::{derive_message_formats, violation};
+
+use crate::checkers::ast::Checker;
+
+#[violation]
+pub struct AssignmentDefaultInStub;
+impl Violation for AssignmentDefaultInStub {
+    #[derive_message_formats]
+    fn message(&self) -> String {
+        todo!("implement message");
```

Comment:
Do you plan to remove this? Or fill it in?

Reply:
This is just an example of what happens when adding a new rule.
All files in this PR except `scripts/add_rule.py` should be removed before merging.